lib/empire_view_model.dart

```diff
@@ -100,13 +100,19 @@ abstract class EmpireViewModel {
   ///late final EmpireProperty<String> name;
   ///late final EmpireProperty<int> age;
   ///
-  ///setMultiple({name: 'Doug', age: 45});
+  ///setMultiple([
```

Comment:
Lets add the restriction that each setter item can only have one key/value pair in the docstring